larray/core/group.py

```diff
@@ -1055,6 +1055,74 @@ def opmethod(self, other):
     __ne__ = _binop('ne')
     __eq__ = _binop('eq')
 
+    # TODO : test also group.axis?
```

Comment:
We should probably have **some** explicit check but I don't know how far we should go. There is already some kind of indirect/implicit check because you use == which uses .eval() which ensures the labels produced are the same (even if the axes are different).

I think the goal of .equals is to check whether or not two groups would return the same values if used on an array (both getitem and aggregates). For that to be true in the presence of ambiguous labels, we need to check the axis name is the same in both groups, or, if we have arrays with several axes with the same name (as is common when using matrices) we have to check for that the axis object is the same. But this pickiness might be annoying in other cases, so I don't really know what's best.

Reply:
we may add an argument `test_axis=True` to Group.equals?

Reply:
I think that by default we should check the name of the axis (if any) but not the object because it would be too confusing if we have:
```python
>>> a = Axis('a=a0..a2')
>>> a2 = Axis('a=a0..a2')
>>> a.equals(a2)
True
>>> # but
>>> a['a0,a2'].equals(a2['a0,a2'])
False
```
because if you don't check the ids of the objects, you don't see that the axes are different.

Unsure having an argument to check that the axis objects are actually the same is worth it. Let's avoid it for now, if that's ok with you.

Reply:
OK